fluent_pages/pagetypes/fluentpage/migrations/0002_add_translation_model.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fluentpage', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FluentPageTranslation',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('language_code', models.CharField(max_length=15, verbose_name='Language', db_index=True)),
+                ('layout_translated', models.ForeignKey(verbose_name='Layout', to='fluent_pages.PageLayout', null=True)),
```

Comment:
Is it necessary to name the field as `layout_translated` first? Perhaps this was needed to generate the migrations, but in practice, I've migrated other apps without having to rename the fields as such.